reference/5.1/Microsoft.PowerShell.Core/About/about_Hash_Tables.md

```diff
@@ -141,84 +161,155 @@ Shape                          Square
+IsSynchronized Property              bool IsSynchronized {get;}
+Keys           Property              System.Collections.ICollection Keys {get;}
+SyncRoot       Property              System.Object SyncRoot {get;}
+Values         Property              System.Collections.ICollection Values {get;}
 ```
 
 ```powershell
-$hash.values
+$dictionary | Get-Member -MemberType Properties, ParameterizedProperty
 ```
 
 ```Output
-1
-Square
-Blue
-```
-
-Each key name is also a property of the hashtable, and its value is the value
-of the key name property. Use the following format to display the property
-values.
-
-```Syntax
-$hashtable.<key>
-<value>
-```
-
-For example:
-
-```powershell
-$hash.Number
-1
-
-$hash.Color
-Blue
-```
-
-hashtables have a **Count** property that indicates the number of key-value
-pairs in the hashtable.
-
-```powershell
-$hash.count
-3
-```
-
-hashtable tables aren't arrays, so you can't use an integer as an index
-into the hashtable, but you can use a key name to index into the hashtable.
-If the key is a string value, enclose the key name in quotation marks.
-
-For example:
-
-```powershell
-$hash["Number"]
-1
-```
+   TypeName: System.Collections.Specialized.OrderedDictionary
+
+Name           MemberType            Definition
+----           ----------            ----------
+Item           ParameterizedProperty System.Object Item(int index) {get;set;},
+                                     System.Object Item(System.Object key) {get;set;}
+Count          Property              int Count {get;}
+IsFixedSize    Property              bool IsFixedSize {get;}
+IsReadOnly     Property              bool IsReadOnly {get;}
+IsSynchronized Property              bool IsSynchronized {get;}
+Keys           Property              System.Collections.ICollection Keys {get;}
+SyncRoot       Property              System.Object SyncRoot {get;}
+Values         Property              System.Collections.ICollection Values {get;}
+```
+
+The most-used properties are **Count**, **Keys**, **Values**, and **Item**.
+
+- The **Count** property that indicates the number of key-value pairs in the
+  object.
+
+- The **Keys** property is a collection of the key names in the hashtable or
+  dictionary.
+
+  ```powershell
+  PS> $hash.Keys
+  three
+  2
+  1
+
+  PS> $dictionary.Keys
+  1
+  2
+  three
+  ```
+
+- The **Values** property is a collection of the values in the hashtable or
+  dictionary.
+
+  ```powershell
+  PS> $hash.Values
+  3
+  two
+  one
+
+  PS> $dictionary.Values
+  one
+  two
+  3
+  ```
+
+- The **Item** property is a parameterized property that returns the value of
+  the item that you specify. Hashtables use the key as the parameter to the
+  parameterized property, while dictionaries use the index by default. This
+  difference affects how you access the values for each type.
+
+## Accessing values
+
+There are two common ways to access the values in a hashtable or dictionary:
+member notation or array notation.
```

Comment:
Nit: Thinking about this, I wonder if it's more semantically accurate to call it _index notation_ - in arrays the index is always an integer, but you can index into a hashtable by key and into an ordered dictionary by key or index.

Reply:
Nit picked.